admin/controllers/school.js

```diff
@@ -36,6 +36,7 @@ controller.getSchoolLandingPage = async (req, res, next) => {
     const currentDate = moment.tz(req.user.timezone || config.DEFAULT_TIMEZONE)
     const resultsOpeningDay = resultPageAvailabilityService.getResultsOpeningDate(currentDate, checkWindowData.checkEndDate)
     const isResultsFeatureAccessible = resultPageAvailabilityService.isResultsFeatureAccessible(currentDate, resultsOpeningDay)
+    // TODO: Determine if the below call will still be needed
```

Comment:
@GuyHarwood @jon-shipley I believe the below call will be redundant on school home page but I'd like to get confirmation before removing it.